src/entry_points/glossary.md

```diff
@@ -22,10 +22,14 @@
 
 - **AggregateTuple**: An AggregateTuple corresponds to the specification of an aggregation task of several model /model updates using an aggregate algo. It leads to the creation of one model / model update.
 
+- **Assets**: Assets are objects that define a ML project, and have to be implemented by an organization before being added in the Substra network. Substra registers, stores and organizes computations on four different kinds of Assets: Objectives, Datasets, Algorithms and, Models. These assets can be private or shared depending on their permission regime.
+
```

Comment:
Would you kindly add some links to the concepts in this document, please?

Reply:
Is a link to the whitepaper sufficient ?